app/controllers/people_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PeopleController < ApplicationController
   def index
+    @people = Person.where(occupation: 'thief')
+    render @people
```

Comment:
* Index Scan: mysql reads 500000 rows from **people** via _index_people_on_occupation_ (occupation).
 * Results: mysql returns 22.0mb (500000 rows) to the client.
 * Estimated query time: 8.72s